candle-nn/src/var_builder.rs

```diff
@@ -137,6 +135,58 @@ impl<'a> VarBuilder<'a> {
 }
 
 impl<'a> VarBuilder<'a> {
+    pub fn get_sharded(
+        &self,
+        tensor_name: &str,
+        dim: usize,
+        rank: usize,
+        world_size: usize,
```

Comment:
It's a bit unclear what `world_size` means here, add a comment or some documention to the function?

Reply:
I updated with a rustdoc on top.

`rank` and `world_size` are definitely names borrowed from torch distributed.
We could rename them to something like `block_id` , `n_blocks` or something along those lines.